problemsets-mat223/modules/module5.tex

```diff
@@ -53,7 +60,22 @@
 	and let $\vec v=\mat{-1\\-1}$. Use the fact that $\vec v-\Proj_\ell\vec v$ is a normal vector to $\ell$
 	to find Find $\Proj_\ell \vec v$.
 
-	XXX Finish
+	Since $\vec v-\Proj_\ell\vec v$ is a normal vector to $\ell$, $\vec v-\Proj_\ell\vec v$ is orthogonal to $\vec d=\mat{1\\1}$. Thus, let $\Proj_\ell\vec v = \mat{x\\y}$ for some unknown $x,y\in\R$, we have
+	\begin{equation}
+	\label{EQREGSYS1}
+	    (\vec v-\Proj_\ell\vec v)\cdot\vec d=0 \implies \mat{-1-x\\-1-y}\cdot \mat{1\\1}=0\implies x+y=-2.
```

Comment:
Put `\qquad`s around `\implies`. Remove the second  implies.

-> `\mat{-1-x\\-1-y}\cdot \mat{1\\1}= x+y=-2`